member.go

```diff
@@ -70,6 +70,17 @@ func (c *Client) Member(nick string) (member *Member, err error) {
 	return
 }
 
+func (c *Client) MemberFromToken(token string) (member *Member, err error) {
```

Comment:
Since consumers are not likely to use multiple tokens at once, I think we should rename this method to simply `.Member()`:

```go
trello, err := trello.NewAuthClient(appKey, &token)
if err != nil {
   return err
}

member, err := trello.Member()
```

What do you think?